### DISCUSSION TITLE
using several pulse audio outputs

### DISCUSSION THREAD

**Discussion by mickael868** [MusicPlayerDaemon/MPD#1620] [Q&A] [answered]
Hi,
I would like to use mpd with 2 audio outputs (2 USB sound cards). 
Everything works fine for the first output if I configure no audio_put block and my client (cantata) shows "default detected output'. If I set the pulseaudio sink names they are not recognized by mpd, the client shows the correct output names ("kitchen" and "bathroom") but they are both routed to the first output.

mpd.conf
```
# Recommended location for database
db_file            "~/.config/mpd/database"

# The music directory is by default the XDG directory, uncomment to amend and choose a different directory
music_directory    "/localdata/rw/music"

# Uncomment to refresh the database whenever files in the music_directory are changed
auto_update "yes"

# Uncomment to enable the functionalities
playlist_directory "~/.config/mpd/playlists"

pid_file           "~/.config/mpd/pid"
state_file         "~/.config/mpd/state"
sticker_file       "~/.config/mpd/sticker.sql"
log_file           "~/.config/mpd/log"

audio_output {
        type            "pulse"
        name            "bathroom"
        device          "alsa_output.usb-GeneralPlus_USB_Audio_Device-00.analog-stereo"
}

audio_output {
        type            "pulse"
        name            "kitchen"
        device          "alsa_output.usb-GeneralPlus_USB_Audio_Device-00.analog-stereo.2"
}

bind_to_address "0.0.0.0"
```

pacmd list-sinks | grep -e 'name:' -e 'index:'

```
    index: 0
        name: <alsa_output.usb-GeneralPlus_USB_Audio_Device-00.analog-stereo>
  * index: 1
        name: <alsa_output.usb-GeneralPlus_USB_Audio_Device-00.analog-stereo.2>
```

relevant logs:
```
config: option 'device' on line 21 was not recognized
config: option 'device' on line 27 was not recognized
```
version : Music Player Daemon 0.23.9 (0.23.9)

I am able to play sound on both sinks using eg. pulsemixer and cmus.

How can I set the proper audio_output devices ?
Thanks

**Comment by MaxKellermann** [comment, marked as answer]
RTFM https://mpd.readthedocs.io/en/stable/plugins.html#pulse

**Comment by mickael868** [comment]
Thanks for your graceful suggestion, I already did read this _fine_ manual but I missed that it is not "device" but "sink"  for a pulse output.
